src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -698,6 +704,77 @@ bool IsArmTemplateFileLike(Uri fileUri)
             }
         }
 
+        private IEnumerable<CompletionItem> GetLocalTestPathCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.TestPath))
```

Comment:
Other than this line, is there any difference between this method and `GetLocalModulePathCompletions`? (Should there be?)

Reply:
I think for now we should only support Bicep files, this code is supporting arm templates too. 
If we only support bicep files, the code might defer from `GetLocalModulePathCompletions`.

Should I make the correction to only support bicep files, or is there any use case so that we can support ARM templates?